addons/website_slides/static/src/js/slides_course_slides_list.js

```diff
@@ -52,12 +52,16 @@ publicWidget.registry.websiteSlidesCourseSlidesList = publicWidget.Widget.extend
      * @private
      */
     _checkForEmptySections: function (){
-        this.$('.o_wslides_js_slides_list_container ul').each(function (){
-            var $emptyCategory = $(this).find('.o_wslides_js_slides_list_empty');
-            if ($(this).find('li.o_wslides_slides_list_slide[data-slide-id]').length === 0) {
-                $emptyCategory.removeClass('d-none').addClass('d-flex');
-            } else {
-                $emptyCategory.addClass('d-none').removeClass('d-flex');
+        var self = this;
+        this.$('.o_wslides_slide_list_category > ul').each(function (){
+            var categoryID = $(this).data('categoryId');
+            var categorySlideCount = $(this).find('.o_wslides_slides_list_slide:not(.o_not_editable)').length;
+            var emptyFlagContainer = self.$('.o_wslides_slide_list_category_header[data-category-id="'+ categoryID +'"] > div:first-child');
+            var emptyFlag = emptyFlagContainer.find('small')[0];
+            if (categorySlideCount === 0 && !emptyFlag){
+                emptyFlagContainer.append('<small class="ml-1 text-muted">Empty</small>');
+            } else if (categorySlideCount > 0 && emptyFlag){
+                emptyFlag.remove();
```

Comment:
I would prefer having the flag in the initial template and toggle a 'd-none' class but it will not work for stable so let's leave it like this :'(